### YamlMime:ManagedReference
items:
- uid: System.Runtime.Remoting.ServerException
  id: ServerException
  children:
  - System.Runtime.Remoting.ServerException.#ctor
  - System.Runtime.Remoting.ServerException.#ctor(System.String)
  - System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ServerException
  nameWithType: ServerException
  fullName: System.Runtime.Remoting.ServerException
  type: Class
  summary: "Eccezione generata per comunicare gli errori al client quando il client si connette ad applicazioni non .NET Framework che non possono generare eccezioni."
  remarks: "ServerException utilizza HRESULT COR_E_SERVER, con valore 0x8013150E.       ServerException utilizza il valore predefinito <xref:System.Object.Equals%2A>implementazione, che supporta l&quot;uguaglianza dei riferimenti.</xref:System.Object.Equals%2A>       Per un elenco di valori di proprietà iniziali per un&quot;istanza di ServerException, vedere i costruttori ServerException."
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class ServerException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Runtime.Remoting.ServerException.#ctor
  id: '#ctor'
  parent: System.Runtime.Remoting.ServerException
  langs:
  - csharp
  name: ServerException()
  nameWithType: ServerException.ServerException()
  fullName: System.Runtime.Remoting.ServerException.ServerException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.Runtime.Remoting.ServerException&quot;> </xref> classe con le proprietà predefinite."
  remarks: "<xref:System.Runtime.Remoting.ServerException>eredita dalla <xref:System.SystemException>classe.</xref:System.SystemException></xref:System.Runtime.Remoting.ServerException> La tabella seguente illustra i valori di proprietà iniziali per un&quot;istanza della classe <xref:System.Runtime.Remoting.ServerException>: | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`|   | <xref:System.Exception.Message%2A>| Una stringa vuota (&quot;&quot;) |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.ServerException>"
  syntax:
    content: public ServerException ();
    parameters: []
  overload: System.Runtime.Remoting.ServerException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Runtime.Remoting.ServerException
  langs:
  - csharp
  name: ServerException(String)
  nameWithType: ServerException.ServerException(String)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.Runtime.Remoting.ServerException&quot;> </xref> classe con un messaggio specificato."
  remarks: "<xref:System.Runtime.Remoting.ServerException>eredita dalla <xref:System.SystemException>classe.</xref:System.SystemException></xref:System.Runtime.Remoting.ServerException> La tabella seguente illustra i valori di proprietà iniziali per un&quot;istanza della classe <xref:System.Runtime.Remoting.ServerException>: | Tipo di proprietà | Condizione |   |-------------------|---------------|   | <xref:System.Exception.InnerException%2A>| `null`|   | <xref:System.Exception.Message%2A>| Il `message` stringa |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.ServerException>"
  syntax:
    content: public ServerException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio che descrive l&quot;eccezione"
  overload: System.Runtime.Remoting.ServerException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Runtime.Remoting.ServerException
  langs:
  - csharp
  name: ServerException(String,Exception)
  nameWithType: ServerException.ServerException(String,Exception)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.Runtime.Remoting.ServerException&quot;> </xref> classe con un messaggio di errore specificato e un riferimento all&quot;eccezione interna che è la causa dell&quot;eccezione corrente."
  remarks: "Un&quot;eccezione generata come risultato diretto di un&quot;eccezione precedente deve includere un riferimento all&quot;eccezione precedente nella <xref:System.Exception.InnerException%2A>proprietà.</xref:System.Exception.InnerException%2A> Il <xref:System.Exception.InnerException%2A>proprietà restituisce lo stesso valore passato al costruttore o `null` se il <xref:System.Exception.InnerException%2A>proprietà non fornisce il valore di eccezione interna al costruttore.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       La tabella seguente illustra i valori di proprietà iniziali per un&quot;istanza della classe <xref:System.Runtime.Remoting.ServerException>: | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Il riferimento all&quot;eccezione interna |   | <xref:System.Exception.Message%2A>| La stringa di messaggio di errore |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.ServerException>"
  syntax:
    content: public ServerException (string message, Exception InnerException);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio di errore che spiega il motivo dell&quot;eccezione."
    - id: InnerException
      type: System.Exception
      description: "Eccezione che è la causa dell&quot;eccezione corrente. Se il `InnerException` parametro non è <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, l&quot;eccezione corrente viene generata in un <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blocco che gestisce l&quot;eccezione interna."
  overload: System.Runtime.Remoting.ServerException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Runtime.Remoting.ServerException.#ctor
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException()
  nameWithType: ServerException.ServerException()
  fullName: System.Runtime.Remoting.ServerException.ServerException()
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String)
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException(String)
  nameWithType: ServerException.ServerException(String)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException(String,Exception)
  nameWithType: ServerException.ServerException(String,Exception)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Runtime.Remoting.ServerException.#ctor*
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException
  nameWithType: ServerException.ServerException
